shared/styles/css.js.flow

```diff
@@ -26,8 +26,9 @@ export type _StylesCrossPlatform = {|
   borderWidth?: number,
   bottom?: Dimension,
   color?: Color,
+  cursor?: 'pointer',
```

Comment:
i don't think these additions are correct -- these are desktop-only styles. So whatever is setting cursor, display, etc. should be using `platformStyles`